### PR TITLE
Add  pending games and originals

### PR DESCRIPTION
**Pull request by pehsa**
Add Open Sonic #886 
Add Simitone #884 
Add OpenVIII #883 
Add Alive_Reversing #880 
Add Dreerally #876 (Death Rally)
Add Gang Garrison 2 #872 (Team Fortress 2)
Add Vega Strike #869 
Add osu! #868 (Osu! Tatakae! Ouendan, Beatmania IIDX, Taiko no Tatsujin; Added DS to platforms)
Add OpenBVE #867 (BVE Trainsim)
Add UnCiv #863 (Civilization V)
Add Magical Life #847 (RimWorld)

### CODE REVIEW

File: games/u.yaml [hunk 1]
```diff
@@ -160,6 +160,35 @@
...
   url: http://jackyjjc.com/
+  
+- name: UnCiv
+  type: remake
+  originals:
+  - Civilization V
+  repo: https://github.com/yairm210/UnCiv
+  url: https://play.google.com/store/apps/details?id=com.unciv.app
+  development: active
+  status: semi-playable
```
Comment (yairm210) [inline games/u.yaml]: Not sure what the standard is, so for the record, it's fully playable on its own but doesn't have all the features of the original

File: games/g.yaml [hunk 2]
```diff
@@ -20,6 +20,29 @@
...
+  originals:
+  - Team Fortress 2
+  repo: https://github.com/Gang-Garrison-2/Gang-Garrison-2/
+  url: https://www.ganggarrison.com/
+  development: sporadic
+  status: playable
+  multiplayer:
+  - Online
+  lang:
+  - GML
```
Comment (cxong) [inline games/g.yaml]:
```suggestion
  - GML
  framework:
  - GameMaker Studio
```

File: games/g.yaml [hunk 3]
```diff
@@ -20,6 +20,29 @@
...
+  license:
+  - MPL
+  content: open
+  info: 2D demake of the Team Fortress 2
```
Comment (cxong) [inline games/g.yaml]:
```suggestion
  info: 2D demake of Team Fortress 2
```

File: games/o.yaml [hunk 4]
```diff
@@ -1588,6 +1648,25 @@
...
+  - Taiko no Tatsujin
+  - Beatmania IIDX
+  repo: https://github.com/ppy/osu
+  url: https://osu.ppy.sh/home
+  development: active
+  status: playable
+  lang:
+  - C#
+  framework:
+  - .Net Core
```
Comment (cxong) [inline games/o.yaml]:
```suggestion
  - .NET Core
```

File: originals/b.yaml [hunk 5]
```diff
@@ -302,3 +314,10 @@
     theme:
     - Management
 
+- name: BVE Trainsim
+  names:
+  - Boso View Express
+  external:
+    wikipedia: BVE Trainsim
+  platform:
+  - Windows
```
Comment (cxong) [inline originals/b.yaml]:
```suggestion
  - Windows
  meta:
    genre:
    - Simulation
    subgenre:
    - Train Simulator
```
Add "Train Simulator" to subgenres schema

File: originals/r.yaml [hunk 6]
```diff
@@ -59,6 +59,15 @@
...
     - Platform
+    
+- name: RimWorld
+  external:
+    wikipedia: RimWorld
+  platform:
+  - Windows
+  meta:
+    genre:
+    - Simulation
```
Comment (cxong) [inline originals/r.yaml]:
```suggestion
    - Simulation
    theme:
    - Sci-Fi
```

File: originals/t.yaml [hunk 7]
```diff
@@ -1,3 +1,23 @@
...
+    genre:
+    - Rhythm
+
+- name: Team Fortress 2
+  names:
+  - TF2
+  external:
+    wikipedia: Team Fortress 2
+  platform:
+  - Windows
```
Comment (cxong) [inline originals/t.yaml]:
```suggestion
  - Linux
  - OSX
  - PlayStation 3
  - Windows
  - Xbox 360
```
Add PlayStation 3 and Xbox 360 to schema

**Pull request merged**